src/data/courseDependenciesData.js

```diff
@@ -3,6 +3,74 @@ import {find} from 'lodash'
 const courseDependencyData = (courseSlug) =>
   find(
     [
+      {
+        id: '425791',
+        type: 'playlist',
+        guid: '93bd',
+        slug: 'declarative-uis-without-css-with-elm-ui-93bd',
+        customOgImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1615288740/next.egghead.io/resources/declarative-uis-without-css-with-elm-ui/share-card-declarative-uis-without-css-with-elm-ui_2x.png',
+        dependencies: {
+          elm: '>=0.19.1',
+        },
+        freshness: {
+          status: `fresh`,
+          title: `This is a Fresh Course`,
+          text: `This course is new and up-to-date. It will provide you with a different paradigm to viewing design, layout, and CSS.`,
```

Comment:
```suggestion
          text: `This course is new and up-to-date. It will provide you with a different paradigm to view design, layout, and CSS.`,
```